living_documentation_regime/model/consolidated_issue.py

```diff
@@ -135,9 +134,9 @@ def project_issue_statuses(self) -> list[ProjectStatus]:
 
     # Error property
     @property
-    def error(self) -> Optional[str]:
+    def errors(self) -> dict[str, str]:
```

Comment:
The method doc string looks old.

Reply:
Commit: [fdcf9aa](https://github.com/AbsaOSS/living-doc-generator/pull/79/commits/fdcf9aafdba5e185c3d742ae773372c5214371e6)